R/utilities-units.R

```diff
@@ -641,3 +643,49 @@ initializeDimensionAndUnitLists <- function() {
 
   return(yData)
 }
+
+
+#' Find the most common units
+#'
+#' @inheritParams .unitConverter
+#' @param ... The name of the column containing units (e.g. `xUnit`).
```

Comment:
Why `...` and not `byColumn` or `unitColumn` or something? I hate `...`

Reply:
I was trying to avoid using embrace operator. 

At any rate, I've changed this now.